test/functional/tag_controller_test.rb

```diff
@@ -570,24 +562,6 @@ def setup
     assert_equal selector.size, 1
   end
 
-  test 'should have a active asked and an inactive answered tab for question' do
-    tag = tags(:question)
-
-    get :show_for_author, params: { id: tag.name, author: 'jeff' }
-
-    selector = css_select '#asked-tab.active'
-    assert_equal selector.size, 1
-    assert_select '#answered-tab', 1
-  end
-
-  test 'should list answered questions' do
```

Comment:
Sorry, you need to remove the test at line 564 of this file and not this one. Thanks

Reply:
No problem ,now it's working fine